Signal/src/ViewControllers/ConversationView/Cells/ConversationMediaView.swift

```diff
@@ -234,7 +234,6 @@ public class ConversationMediaView: UIView {
         AssertIsOnMainThread()
 
         guard let loadBlock = loadBlock else {
-            owsFailDebug("Missing loadBlock")
```

Comment:
Previously, this view assumed that it was only used to render `TSAttachmentStream`.  Now it's also used for `TSAttachmentPointer` (e.g. un-downloaded incoming attachments).